03/app.js

```diff
@@ -1,27 +1,65 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { v4 as uuid } from 'uuid';
 
 import Category from './Category';
 import Cart from './Cart';
+import Product from './Product';
 
 import data from './data.json';
 
 class App extends React.Component {
-    state = {
-        cart: [],
-    }
-    
-    render() {
-        return (
-            <section>
-                <Category />
-                <Cart />
-            </section>
-        )
-    }
+  state = {
+    cart: [],
+  };
+
+  addToTheCart = (product) => {
+    const cartCopy = [...this.state.cart];
+    product.isCart = true;
+
+    cartCopy.push(product);
+    this.setState({ cart: cartCopy });
```

Comment:
Chyba ładniej by było `this.setState({ cart: [...this.state.cart, {...product, isCart: true}]}`